Tasks/FileTransformV1/task.json

```diff
@@ -11,15 +11,14 @@
         "Build",
         "Release"
     ],
-    "preview": "true",
     "runsOn": [
         "Agent",
         "DeploymentGroup"
     ],
     "version": {
         "Major": 1,
-        "Minor": 1,
-        "Patch": 6
+        "Minor": 153,
```

Comment:
👍 , please do share this change with the team.

Reply:
Will do